### ISSUE TITLE
vgg16c truncated mean at 94%

### ISSUE THREAD

**Issue by echurch** [drinkingkazu/ArchitecturePaperStudy#2]
Please continue training

![image](https://cloud.githubusercontent.com/assets/8322304/22639448/059e5d04-ec03-11e6-8af5-7c1c748fe3ec.png)

**Comment by drinkingkazu** [comment]
OK I would give a priority to vgg16b over vgg16c. This ticket won't be closed till training is finished.

**Comment by echurch** [comment]
Despite claims, vgg16c is either not running or pubs isn't updating on its output. Anyone know which it is?

**Comment by drinkingkazu** [comment]
Thanks for reminding! yes I forgot to register for inference! will do now

**Comment by echurch** [comment]
I'd do it on dgx-1.pnnl.gov, but Kevin's got 2 networks there that have priority

**Comment by drinkingkazu** [comment]
? I said I forgot to register for inference. training is running. I'm rsyncing weight now so we have to wait a bit for inference to start.

**Comment by echurch** [comment]
ok, sorry for my confusing comment. I note wu continues to not update vgg16c. Maybe you were waiting for me to confess to confusion before re-starting it.

**Comment by echurch** [comment]
Ok, here's latest rsync, then Epoch*.ipynb analysis. I don't understnad why the max epoch is 189 here, even though the plotterdata at wu shows we're up over 205 epochs. Does anyone? Could there be a discontinuity in the training, or something? The drinkingkazu_sp_vgg16c.test,train,log files don't indicate that....

**Comment by echurch** [comment]
![image](https://cloud.githubusercontent.com/assets/8322304/23494661/0179b592-fec9-11e6-87d8-1767390c539f.png)

**Comment by drinkingkazu** [comment]
Yes! this is due to Kazu's fault. because vgg16c's weights are large, I have not rsync-ed everything. I wanted to see if we need more. Looks like we do (sigh). I'll rsync now.

**Comment by echurch** [comment]
okay, lets stop this one, k?

**Comment by echurch** [comment]
![image](https://cloud.githubusercontent.com/assets/8322304/23542107/2bbfbe46-ffa0-11e6-81b3-8d5fe1705c89.png)
